requirements/base.in

```diff
@@ -21,9 +21,8 @@ django-better-admin-arrayfield
 djangorestframework
 django-filter~=21.1
 djangorestframework-camel-case
-drf-yasg
-vng-api-common
-
+drf_spectacular
+-e git+https://github.com//MatthijsBekendam/vng-api-common.git@drf_compatibilty_newer_django_fix#egg=vng_api_common
```

Comment:
Deze dependency wil je als `non-editable` installeren, dit zorgt er ook voor dat er geen wijzigingen nodig zijn in de `Dockerfile`